engine/dlib/src/dmsdk/dlib/profile.h

```diff
@@ -587,7 +587,7 @@ namespace dmProfile
     /// Internal, do not use.
     struct ProfileScope
     {
-        inline ProfileScope(const char* name, uint64_t* name_hash)
+        inline ProfileScope(const char* name, uint64_t* name_hash) : valid(0)
```

Comment:
Another issue. The scope might be accidentially "valid" due to garbage in the `valid` flag.